Formula/libgrape-lite.rb

```diff
@@ -0,0 +1,62 @@
+class LibgrapeLite < Formula
+  desc "C++ library for parallel graph processing"
+  homepage "https://github.com/alibaba/libgrape-lite"
+  url "https://github.com/alibaba/libgrape-lite/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "aafc8e5e9b122c4bde4370ee3f2ab8afb043f2a1547cacc062991ad420e3b9a4"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+
+  depends_on "glog"
+  depends_on "open-mpi"
+
+  on_linux do
+    depends_on "gcc"
+  end
+
+  def install
+    ENV.cxx11
+
+    system "cmake", "-S", ".", "-B", "build",
+                    "-DCMAKE_CXX_STANDARD=11",
```

Comment:
Is this needed along the ENV.cxx11?

Reply:
Building this library requires c++11. Doesn't `Env.cxx11` means c++11?

Reply:
I have delete the `ENV.cxx11` line.

Reply:
Yes, but do you still need to tell CMake separately?

Reply:
I would use `ENV.cxx11` and remove the `-DDCMAKE_CXX_STANDARD=11` lines.

Reply:
Fixed.